lib/modules/apostrophe-login/index.js

```diff
@@ -349,12 +353,24 @@ module.exports = {
       return next();
     };
 
+    // Check if the first character about the option `loginUrl` is `/`
+    // and if is not present, add the `/` in the first position.
+    // If the option is not present, return the default url `/login`
+    self.fetchLoginUrl = function() {
+      if (self.options.loginUrl) {
+        return self.options.loginUrl && self.options.loginUrl.startsWith('/') ? self.options.loginUrl : `/${self.options.loginUrl}`
+      }
+      else {
+        return '/login'
+      }
+    }
+
     // Add the `/login` route, both GET (show the form) and POST (submit the form).
     // Also add the `/logout` route.
 
     self.addRoutes = function() {
       if (self.options.localLogin !== false) {
-        self.apos.app.get('/login', function(req, res) {
+        self.apos.app.get(self.fetchLoginUrl(), function(req, res) {
```

Comment:
This URL appears in many other places: for CSRF protection, for redirects from other steps and for other things. You will need to switch the code to call getLoginUrl in each case.